client/src/Game/Home.js

```diff
@@ -149,7 +153,7 @@ class Home extends Component {
             }
             console.log(this.state.pendingMatches[index].gameID);
             request(this.state.pendingMatches[index].gameID,"retrieveMatch").then(gameState => {
-                this.setGame("pendingMatches", index, gameState);
+                this.setGame("pendingMatches", index, gameState, ID, "Active");
```

Comment:
Should this be "Pending"?

Reply:
That doesn't need to be pending, because a pending game won't ever be displayed. An accepted invitation will turn into an active game immediately.